pkg/tso/keyspace_group_manager.go

```diff
@@ -513,6 +506,7 @@ func (kgm *KeyspaceGroupManager) watchKeyspaceGroupsMetaChange(revision int64) (
 func (kgm *KeyspaceGroupManager) isAssignedToMe(group *endpoint.KeyspaceGroup) bool {
 	// If the default keyspace group isn't assigned to any tso node/pod, assign it to everyone.
 	if group.ID == mcsutils.DefaultKeyspaceGroupID && len(group.Members) == 0 {
+		group.Members = []endpoint.KeyspaceGroupMember{{Address: kgm.tsoServiceID.ServiceAddr}}
```

Comment:
It's kind of tricky to me here, can't we use some more "legitimate" ways to achieve this?

Reply:
Agree. If the the default keyspace group's member list is empty, we should fall back the single-keyspace-group like before, and let balancer to add member list -- this is the more "legitimate" way. 

Removed this piece of hack and change to the above logic.

Reply:
And I need to wait hanyang's https://github.com/tikv/pd/pull/6342 and the refactor for common watching (a range) module to thoroughly solve this issue.

Reply:
even after https://github.com/tikv/pd/pull/6342 is committed, I want to promot this logic "if the member list of the default keyspace group is empty, we treat it as full tso node list" for two purposes: